libs/journeys/ui/src/components/StepFooter/ChatButtons/ChatButtons.stories.tsx

```diff
@@ -17,16 +17,17 @@ const ChatButtonsDemo: Meta<typeof ChatButtons> = {
   title: 'Journeys-Ui/StepFooter/ChatButtons'
 }
 
-type Story = StoryObj<{ chatButtons: ChatButton[] }>
+type Story = StoryObj<{ chatButtons: ChatButton[] } & { website: boolean }>
```

Comment:
```suggestion
type Story = StoryObj<{ chatButtons: ChatButton[], website: boolean }>
```
Any reason why we couldn't do this?